PlaceRouteHierFlow/PnRDB/ReadLEF.cpp

```diff
@@ -137,7 +137,7 @@ void PnRdatabase::_ReadLEF(istream& fin, const string& leffile) {
           if (temp[1].front() == 'M') {
             interMetals.resize(interMetals.size() + 1);
             interMetals.back().metal = temp[1];
-          } else if (temp[1].front() == 'V' && temp[1].back() != '0') {
+          } else if (temp[1].front() == 'V' && temp[1].back() != '0' && temp[1].back() != 'T') {
```

Comment:
@Lastdayends , let's not hard-code any layer names. This issue is beyond VT. Please see my last commit with a test replicating this issue with V1.

Reply:
@soneryaldiz 

I mean VT is beyond V1-Vx.

Reply:
@Lastdayends , first, please take a look at `test_ru_m1m2_v`. This test includes M1, V1 and M2. IT appears V1 is considered as an obstruction when it should not really.